content/en/releases/v0.14.0.md

```diff
@@ -0,0 +1,195 @@
+---
+title: v0.14.0
+date: 2023-08-18T00:00:00+01:00
+version: 0
+subversion: 14
+extraversion: 0
+codename: Atlas
+commit: TODO
```

Comment:
This will be updated after release I guess?

Reply:
Yes, I need the last commit ID, still waiting for some work to get done.